source/scripts/Background/controllers/MainController.ts

```diff
@@ -1648,6 +1652,7 @@ const MainController = (walletState): IMainController => {
     ...keyringManager,
     openDAppErrorModal,
```

Comment:
This is the method I mentioned https://github.com/syscoin/pali-wallet/pull/602/files#r1695634367